tests/std/tests/P1502R1_standard_library_header_units/test.cpp

```diff
@@ -176,12 +176,8 @@ int main() {
         const to_chars_result result = to_chars(buf, end(buf), 3.14);
         assert(result.ec == errc{});
         assert(result.ptr == end(buf));
-#ifdef MSVC_INTERNAL_TESTING // TRANSITION, DevCom-1224512 (char_traits)

```

Comment:
I did a double take after reading this as `MSVC_INFERNAL_TESTING`.

Reply:
This has definitely found a bunch of infernal compiler errors. :smiling_imp: